integrations/zoom.js

```diff
@@ -78,6 +79,7 @@ module.exports = async function(robot, kredits) {
 
     for (const displayName of names) {
       await createContributionFor(displayName, meetingDetails);
+      await sleep(60000); // potentially to prevent too many transactions at the sametime. transactions need to be ordered because of the nonce. not sure though if this is needed.
```

Comment:
I thought this is exactly what the nonce management would solve. We do wait for that long in all the other integrations and never had an issue with nonces as a result (only with warnings, but not with missing contribution records).

Reply:
yeah, the NonceManager keeps a track of the nonces... 
normally ethers.js tries to get the pending transactions count to calculate the nonce before signing it. This can fail if more transactions are created in a short time. The NonceManager should solve this by keeping track of the number of created tranactions. 

I just wanted to be super save here for the next deploy that it works. thus I kept that sleep in there.